src/db/spanner/models.rs

```diff
@@ -566,15 +566,16 @@ impl SpannerDb {
                 "ids".to_owned(),
                 as_list_value(uncached.into_iter().map(|id| id.to_string())),
             );
-            let result = self
+            let rs = self
```

Comment:
later on, you use `affected_rows`. We might want to use a better var name here as well, say `collection_row` or something.

Reply:
I'm using rs for ResultSet (from execute) whereas the new execute_dml now returns the affected_rows